### PR TITLE
Modify docs to note auto disarm is now the default

### PR DESCRIPTION
**Pull request by hamishwillee**
This also splits the getting started level takeoff and landing docs into separate headings.

Fyi @MaEtUgR - does this look OK?

### CODE REVIEW

File: en/config/safety.md [hunk 1]
```diff
@@ -134,16 +134,17 @@ Loiter Time | [RTL_LAND_DELAY](../advanced_config/parameter_reference.md#RTL_LAN
...
+For Multicopters (only) you can additionally set the descent rate.
 
 ![Safety - Land Mode Settings (QGC)](../../images/qgc/setup/safety_land_mode.png)
 
 The settings and underlying parameters are shown below:
 
 Setting | Parameter | Description
 --- | --- | ---
-Disarm After | [COM_DISARM_LAND](../advanced_config/parameter_reference.md#COM_DISARM_LAND) | Select checkbox to specify that the vehicle will disarm after landing, and enter delay after landing before disarming (must be non-zero). 
+Disarm After | [COM_DISARM_LAND](../advanced_config/parameter_reference.md#COM_DISARM_LAND) | Select checkbox to specify that the vehicle will disarm after landing, and enter delay after landing before disarming (must be non-zero).
```
Comment (MaEtUgR) [inline en/config/safety.md]:
```suggestion
Disarm After | [COM_DISARM_LAND](../advanced_config/parameter_reference.md#COM_DISARM_LAND) | Select checkbox to specify that the vehicle will disarm after landing, and enter delay after landing before disarming. The value must be non-zero but can be a fraction of a second.
```

File: en/flight_modes/land.md [hunk 2]
```diff
@@ -13,27 +14,31 @@ The specific behaviour for each vehicle type is described below.
...
+The vehicle will land at the location at which the mode was engaged.
+The vehicle descends at the rate specified in `MPC_LAND_SPEED` and will disarm after landing (by default).
 
 Landing is affected by the following parameters:
 
 Parameter | Description
 --- | ---
 [MPC_LAND_SPEED](../advanced_config/parameter_reference.md#MPC_LAND_SPEED) | The rate of descent during landing. This should be kept fairly low as the ground conditions are not known.
-[COM_DISARM_LAND](../advanced_config/parameter_reference.md#COM_DISARM_LAND) | Time-out for auto disarm after landing. By default this is 0 (vehicle will not auto-disarm after landing).
+[COM_DISARM_LAND](../advanced_config/parameter_reference.md#COM_DISARM_LAND) | Time-out for auto disarm after landing, in seconds. If set to -1 the vehicle will not disarm on landing.
```
Comment (MaEtUgR) [inline en/flight_modes/land.md]:
```suggestion
[COM_DISARM_LAND](../advanced_config/parameter_reference.md#COM_DISARM_LAND) | Time-out for auto disarm after landing, in seconds. If set to -1 the vehicle will not disarm on landing. Can be a fraction of a second.
```

File: en/flying/basic_flying.md [hunk 3]
```diff
@@ -19,22 +19,27 @@ This will start the propellers on a multicopter.
...
-
-## Takeoff and Landing
+<span id="takeoff-and-landing"></span>
+## Takeoff
 
 The easiest way to takeoff is to use the automatic [Takeoff mode](../flight_modes/takeoff.md) (remembering that you need to arm the vehicle before you can engage the vehicle motors).
-To land again automatically you can use [Land](../flight_modes/land.md) or [Return](../flight_modes/return.md) modes.
 
-> **Tip** The automatic takeoff/landing modes are highly recommended, in particular for Fixed Wing vehicles. 
+Multicopter (and VTOL in multicopter mode) pilots can take off *manually* by enabling [position mode](../flight_modes/README.md#position_fw), arming the vehicle, and then raising the throttle stick above 62.5%.
```
Comment (MaEtUgR) [inline en/flying/basic_flying.md]:
```suggestion
Multicopter (and VTOL in multicopter mode) pilots can take off *manually* by enabling [position mode](../flight_modes/README.md#position_mc), arming the vehicle, and then raising the throttle stick above 62.5%. 
```

**Comment by hamishwillee** [comment]
@MaEtUgR Thanks for the review - and in particular for all the typo fixes. Probably rushing too much on too many things.

**Pull request merged**